python-api-examples/speech-recognition-from-microphone.py

```diff
@@ -116,6 +211,9 @@ def main():
             if last_result != result:
                 last_result = result
                 print("\r{}".format(result), end="", flush=True)
+            is_endpoint = recognizer.is_endpoint(stream)
+            if is_endpoint:
+                recognizer.reset(stream)
```

Comment:
I think current implementation did not handle `Reset` for context biasing properly, you have to add some C++ code to `Reset` in `OnlineRecognizer` to make the contexts biasing work after recognizer being reset.  I mean, the context biasing can not work any more after resetting, you can try it.

Here are some reference code piece:
https://github.com/k2-fsa/sherpa-onnx/blob/c88e672733efe4ec6b345d4cdc6c9d65c07495a8/sherpa-onnx/csrc/online-recognizer.cc#L173-L186

Reply:
@pkufool  Thanks for explanations. I did not think too much about relations between `Reset` and contexts biasing. Will delete this part.

Reply:
You can reset here,  I mean there are some issues for current `recognizer.reset()`,  hope that you can help to fix it.

Reply:
I will fix this in next PR. Since my current knowledge of context biasing logic is not that solid.